src/index.js

```diff
@@ -95,7 +95,6 @@ const entrypoint = function (options = {}) {
 			builder.rimraf(publicDir);
 
 			builder.log.minor(logRelativeDir('Writing client application to', publicDir));
-			builder.writeStatic(publicDir);
```

Comment:
Is this replaced with anything? How are these files now being written?

Reply:
Hmm... I'm just experimenting with the adapter, but stuck on this error. Any guidance? Just wait for this to be resolved? pin to an earlier version of SvelteKit? Thoughts? Thanks.

Reply:
It doesn't look like it was replaced with anything else on the adapter-vercel
https://github.com/sveltejs/kit/commit/819172065cdda6c695f3fb48e60fa451c1caaae6

I'll give this a try.

Reply:
According to [PR5618](https://github.com/sveltejs/kit/pull/5618):
> Have writeClient write both the client and static files and remove writeStatic. This is a breaking change for adapter authors.

`writeClient` does job for `writeStatic` so I removed `writeStatic`.